_config.yml

```diff
@@ -31,7 +31,6 @@ themeColor: purple # green, blue, orange, purple, grey
 fixedNav: 'true' # true or false
 
 permalink: /:year/:month/:title
-markdown: kramdown
```

Comment:
What purpose does this have to be removed? Jekyll uses Kramdown for markdown right now, but it could change in the future.

Reply:
It was in there twice